<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpForm</class>
 <widget class="QWidget" name="HelpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbBack">
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/katynko/svg/go-previous.svg</normaloff>:/katynko/svg/go-previous.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbForward">
       <property name="text">
        <string>Forward</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/katynko/svg/go-next.svg</normaloff>:/katynko/svg/go-next.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="goHome">
       <property name="text">
        <string>Home</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/katynko/svg/go-home.svg</normaloff>:/katynko/svg/go-home.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbFontDec">
       <property name="text">
        <string>Minus</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/katynko/svg/format-font-size-less.svg</normaloff>:/katynko/svg/format-font-size-less.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbFontNorm">
       <property name="text">
        <string>Normal</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/katynko/svg/format-font-size-normal.svg</normaloff>:/katynko/svg/format-font-size-normal.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbFontInc">
       <property name="text">
        <string>Plus</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/katynko/svg/format-font-size-more.svg</normaloff>:/katynko/svg/format-font-size-more.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbLang"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PDF</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/katynko/svg/application-pdf.svg</normaloff>:/katynko/svg/application-pdf.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vltWeb"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_27">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Case Sensitively</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFindPteLog_2">
       <property name="placeholderText">
        <string>Search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_4">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_6">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_30">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
